src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -564,11 +567,29 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </AllowedReferenceAssemblyFileExtensions>
 
     <!--
+        These are the extensions that reference resolution will consider when looking for debug files related
+        to resolved references, for .pdb files
+        -->
+    <AllowedReferenceRelatedDebugFileExtensions Condition="'$(DebugFileExt)' == '.pdb'">
+        .pdb
```

Comment:
can you replace the literal with `$(DebugFileExt)`?

As an overall comment, these .pdb / .mdb string literals appear too many times. Could you extract them out more into constants, so that you define the literal once (or as few times as possible) into a property and then use the properties instead the literals?